base/ltcmd.dtx

```diff
@@ -2530,6 +2622,47 @@
   }
 %    \end{macrocode}
 %
+% Optimised functions needs things done a bit differently as we need to
+% reconstruct the argument spec.
+%    \begin{macrocode}
+\cs_new_protected:Npn \@@_show_optimized:N #1
+  {
+    \exp_args:Nc \@@_show_optimized:NN
+      { \cs_to_str:N #1 \c_space_tl code }
+      #1
+  }
+\cs_new_protected:Npn \@@_show_optimized:NN #1#2
+  {
+    \cs_set:Npe \@@_show_optimized_aux:N ##1
+      {
+        \c_space_tl \c_hash_str ##1 :
```

Comment:
To be consistent with other cases. It's at line beginning here hence leading `~`s are always skipped. 
https://github.com/latex3/latex2e/blob/0f40cbb8ea0c43592db9d1c96dbb7575166b4a7b/base/ltcmd.dtx#L2736-L2740
```suggestion
        \c_space_tl \c_space_tl \c_hash_str ##1 :
```

Reply:
Changed (done as a commit by me as also needs `.tlg` changes).